README.md

```diff
@@ -78,6 +78,7 @@ class CreateVehicleFlow extends FormFlow {
 			array(
 				'label' => 'wheels',
 				'type' => $this->formType,
+				'options' => array(...), //form options
```

Comment:
Please revert this change. The docs need to be updated of course, but this is not enough. I'll take care later.